en/faq.md

```diff
@@ -64,36 +64,30 @@ Check out [C++/Contributing/Plugins](cpp/contributing/plugins.md).
+We have sometimes moved functionality from plugins to the core if it was exposed in multiple plugins. As an example mission upload/download has been moved to the core so it can be used in the Mission plugin as well as the MissionRaw and Geofence plugins.
 
 ### Can MAVSDK run on an embedded platform / microcontroller?
 
-MAVSDK is generally written a bit higher level, geared towards ARM devices such as a Raspberry Pi, smartphone or faster/better.
+MAVSDK is generally written at a bit higher level, geared towards ARM devices such as a Raspberry Pi, a smartphone, or a similar device with equal/better performance.
 
-As it doesn't actually use too much compute, it could in theory be run on a microcontroller such as an ARM Cortex M4, however, it would require the POSIX APIs for serial and networking communication as well as the C++ standard library for things like `std::thread` or `std::vector`.
+As MAVSDK does not actually require too much CPU power, it could in theory be run on a microcontroller such as an ARM Cortex M4, however, it would require the POSIX APIs for serial and networking communication, as well as the C++ standard library for classes/functions such as `std::thread` or `std::vector`.
 
-The recommendation for a microcontroller would be to use the pure [C MAVLink headers](https://mavlink.io/en/mavgen_c/).
+The recommendation for usage with a microcontroller would be to use the pure [C MAVLink headers](https://mavlink.io/en/mavgen_c/).
 
 ### Why is MAVLink Passthrough only available in C++?
 
-The C++ MAVLink passthrough plugin basically exposes the direct C MAVLink API.
+The C++ MAVLink passthrough plugin basically exposes the direct C MAVLink API. While it would be nice to have access to all MAVLink messages in the language wrappers, there are some technical challenges:
 
-While it would be nice to have access to all MAVLink messages in the language wrappers, there are some technical challenges:
 - Essentially, it would mean that all MAVLink APIs would have to be duplicated by the [proto APIs](https://github.com/mavlink/MAVSDK-Proto/tree/main/protos).
   This would increase the API and code size considerably.
 - Alternatively, the API could be exposed without types, using some sort of runtime access like `get_message("ATTITUDE").get_field("pitch").as_float()`.
   This would come with the cost of no type safety and runtime overhead for parsing the strings.
 
 From a MAVSDK project point of view, there is also an advantage of not having a passthrough available in language wrappers; it encourages that required features are contributed back to the open-source project, rather than implemented in private using passthrough, and thus benefitting everyone.
 
-### Why is MAVSDK written in C++?
-
-The requirements for MAVSDK were that it can efficiently run in embedded setups, e.g. as part of an app or onboard a drone on the companion computer which suggested C or C++.
```

Comment:
Yes, it is good to delete the heading. But I would merge the content into the other topic.